h2o-r/h2o-package/R/models.R

```diff
@@ -3209,6 +3215,14 @@ h2o.partialPlot <- function(object, data, cols, destination_key, nbins=20, plot
       } else {
         plot(pp[,1:2], type = "l", main = attr(x,"description") )
       }
+      if(!is.null(save_to)){
```

Comment:
Maybe the documentation should say the file will only get written if plot==TRUE

Reply:
Fixed. Thank you.